roles/shiftstack/templates/pvc.yml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ cifmw_shiftstack_client_pod_name }}-pvc
```

Comment:
I would use set a fixed name so it can be known for all the shiftstackclient pods running on the setup.

Reply:
Do you want both pods to have the same pvc?